### ISSUE TITLE
Why it isnt on Chrome WebStore?

### ISSUE THREAD

**Issue by inson1** [Darkempire78/Github1s-Extension#15]
https://chrome.google.com/webstore/category/extensions

**Comment by clementwzk** [comment]
I did not publish it on Chrome because I do not have a chrome dev account

**Comment by clementwzk** [comment]
Btw, now I can pay for it, but all my extensions are olds... Perhaps in the future if I want to create a new extension

**Comment by inson1** [comment]
> Btw, now I can pay for it, but all my extensions are olds... Perhaps in the future if I want to create a new extension

So does this extension still works? 
If yes then I can spare 5$ 🤔

**Comment by clementwzk** [comment]
> > Btw, now I can pay for it, but all my extensions are olds... Perhaps in the future if I want to create a new extension
> 
> So does this extension still works? If yes then I can spare 5$ 🤔

Yes it should still works! You are right, I will publish them, it's only 5€ lol

**Comment by inson1** [comment]
> > > Btw, now I can pay for it, but all my extensions are olds... Perhaps in the future if I want to create a new extension
> > 
> > So does this extension still works? If yes then I can spare 5$ 🤔
> 
> Yes it should still works! You are right, I will publish them, it's only 5€ lol

5$

Yea Its even cheaper than Google Play. But you can only have 20 extensions, for Google Play you dont have this kind of limitations. But yea 20 extension is still big number for one person.

Well I still refund you and buy you coffee if it still works.
